static/css/components/header-bar.less

```
@@ -441,12 +441,25 @@
 
     .search-bar-submit {
       background: url(/static/images/search-icon.svg) center no-repeat;
-      width: 45px;
+      width: 35px;
       height: 45px;
       border-top-left-radius: .3em;
       border-bottom-left-radius: .3em;
     }
 
+    .vertical-separator {
+      border-left: 1px solid @grey;
```

Make this a little fainter.

```suggestion
      border-left: 1px solid @light-grey;
```